modules/vmdatasource/src/xmpdatasource.cpp

```diff
@@ -21,6 +21,11 @@
 #include "xmpschemasource.hpp"
 #include "xmpmetadatasource.hpp"
 
+#include <XMP.incl_cpp>
+#include <XMP.hpp>
+
+#include "libbase64++.h"
```

Comment:
consider using base64 enc/dec from the XMP sources instead of an external one...